unattended-upgrade

```diff
@@ -1447,8 +1440,12 @@ def calculate_upgradable_pkgs(cache,             # type: apt.Cache
                            pkgs_kept_back,
                            cache,
                            allowed_origins,
-                           blacklisted_pkgs,
-                           whitelisted_pkgs)
+                           blacklist,
+                           whitelist)
+
+    if (cache.get_changes()):
```

Comment:
(nitpick) No need for the extra `()`, i.e. `if cache.get_changes():` is fine :)

Reply:
Fixed that, thanks!